concent_api/core/tests/test_unit_validation.py

```diff
@@ -441,3 +447,212 @@ def test_that_unsupported_format_raises_concent_validation_error(self):  # pylin
         with pytest.raises(ConcentValidationError) as exception_wrapper:
             validate_blender_output_format(unsupported_format)
         assert_that(exception_wrapper.value.error_code).is_equal_to(ErrorCode.MESSAGE_VALUE_NOT_ALLOWED)
+
+
+class TestValidateSceneResolution:
+
+    @pytest.mark.parametrize(
+        'resolution', [
+            '400x400',
+            [0, 0],
+            [100, 0],
+            [0, 100],
+            [-100, 100],
+            [100, -100],
+            ['100', 100],
+            [],
+            ['400x800'],
+            [71830.23, 1000],
+        ]
+    )  # pylint: disable=no-self-use
+    def test_that_invalid_resolution_value_raise_exception(self, resolution):
+        with pytest.raises(ConcentValidationError) as exception_wrapper:
+            validate_resolution(resolution)
+        assert_that(exception_wrapper.value.error_code).is_equal_to(ErrorCode.MESSAGE_INVALID)
+
+    @pytest.mark.parametrize(
+        'resolution', [
+            [1, 1],
+            [100000000, 100000000],
+        ]
+    )  # pylint: disable=no-self-use
+    def test_that_valid_resolution_value_doesnt_raise_exception(self, resolution):
+        try:
+            validate_resolution(resolution)
+        except Exception as exception:  # pylint: disable=broad-except
+            assert False, f"Unexpected exception has been raised: {str(exception)}"
+
+
+class TestValidateSceneUseCompositing:
+
+    @pytest.mark.parametrize(
+        'use_compositing', [
+            'True',
+            1,
+            [True]
+        ]
+    )  # pylint: disable=no-self-use
+    def test_that_invalid_use_compositing_value_raise_exception(self, use_compositing):
+        with pytest.raises(ConcentValidationError) as exception_wrapper:
+            validate_use_compositing(use_compositing)
+        assert_that(exception_wrapper.value.error_code).is_equal_to(ErrorCode.MESSAGE_INVALID)
+
+    @pytest.mark.parametrize(
+        'use_compositing', [
+            True,
+            False
+        ]
+    )  # pylint: disable=no-self-use
+    def test_that_valid_use_compositing_value_doesnt_raise_exception(self, use_compositing):
+        try:
+            validate_use_compositing(use_compositing)
+        except Exception as exception:  # pylint: disable=broad-except
```

Comment:
This is too broad. You should expect a specific exception type here.

Reply:
I think that we should catch here all exception because we assume that this test should pass without any error, but as you suggested I changed to `ConcentValidationError`